tests/garage/torch/algos/test_loss_function_utils.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import pytest
+import tensorflow as tf
+import torch
+
+import garage.tf.misc.tensor_utils as tf_utils
+import garage.torch.algos.loss_function_utils as torch_loss_utils
+from tests.fixtures import TfGraphTestCase
+
+
+def stack(arr, d):
```

Comment:
how does this differ from np.stack?

Reply:
np.stack join a sequence of arrays, and this is doing kind of copying the array d times and join the arrays.